### PR TITLE
INTYGFV-15928: Display banner instead of modal for OriginNormalWarnin…

### PR DESCRIPTION
**Pull request by ChrilleSigerhed**
…g in angular.

### CODE REVIEW

**Comment by ChrilleSigerhed** [comment]
image

**Review by mhornfeldt — APPROVED** [review]
Tests are missing but since it's Angular maybe it was intentional

File: web/src/main/resources/META-INF/resources/webjars/common/webcert/components/wcOriginNormalWarning/wcOriginNormalWarning.directive.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+ */
+angular.module('common').directive('wcOriginNormalWarning',
+    ['$uibModal', '$window', 'common.UserModel', 'common.featureService',
+      function($uibModal, $window, UserModel, featureService) {
+        'use strict';
+        if (!UserModel.user.valdVardgivare) {
+          return;
+        }
+        var shouldOriginNormalWarningDialogBeDisplayed = function() {
+          return UserModel.user.origin === 'NORMAL' &&
```
Comment (mhornfeldt) [inline web/src/main/resources/META-INF/resources/webjars/common/webcert/components/wcOriginNormalWarning/wcOriginNormalWarning.directive.js:27]: Is there a constant to use for normal?

**Comment by ChrilleSigerhed** [comment]
> Tests are missing but since it's Angular maybe it was intentional

No tests class were related to the modal so I decided not to implement a new one. Will look into implementing it.

File: web/src/main/resources/META-INF/resources/webjars/common/webcert/components/wcOriginNormalWarning/wcOriginNormalWarning.directive.js (hunk 1 above)
Comment (ChrilleSigerhed) [reply web/src/main/resources/META-INF/resources/webjars/common/webcert/components/wcOriginNormalWarning/wcOriginNormalWarning.directive.js:27]: I checked, but could not fine one.

**Pull request merged**